00001-enarx-sprint-framework/README.md

```diff
@@ -0,0 +1,542 @@
+that means the PR is still in review and likely just needs a bit more review
+time. Thus, it should stay as-is in the Under Review column.
+
+Beyond that, the rest of the meeting should be dedicated to demos or other
+high-level discussion about work accomplished.
+
+Retrospectives should take a full hour. As detailed in [Planning](#planning),
+the two meetings should be combined, with Retrospective happening first,
+replacing the standup of the first day of a new sprint.
+
+Here's a proposed schedule:
+
+- 20 min: review issues/PRs from last sprint
+- 40 min: demos or other discussion
+
+## Pull Requests
+
+The meetings above detail how we organize and plan our work with issues. This
+section will elaborate how the team should execute that work with pull requests.
+
+As described [above](#tracking-work), pull requests encapsulate the end of a
+task. Pushing them across the finish line, whether that means providing review
+or addressing those reviews, should always be a priority. As such, it's
+important to communicate who needs to take action on a PR at any given time. The
+Enarx project's current workflow does not do this well, and it's an obvious area
+for improvement.
+
+### Workflow
+
+Here's a flowchart of the new and improved proposed workflow for a pull request,
+starting with a PR's submission and going all the way to its merge:
+
+![PR workflow diagram](./pr-workflow.svg)
+
+As with every issue, every PR will be reviewed through a Triage, during which
+reviewers will be requested and assigned. At least two reviewers must be
+assigned for review, though the PR author may optionally request other people
+review it without explicitly assigning them. These "optional reviewers" may
+choose not to review it; however, if they _do_ submit a review, they are
+expected to follow through the entire process until the PR is merged.
+
+Note that the flowchart follows the flow for an individual reviewer and the
+author; this process is replicated for every individual reviewer involved with
+the PR.
+
+The cycle starts with the `reviewer_assigned` state, which indicates that it's
+the reviewer's responsibility to submit a PR review.
+
+Once the reviewer submits a review, two things can happen based on the content:
+
+- If the review approves of the PR, it moves into the `approved` state for that
+  individual author. From here, the PR waits on other reviewers until it meets
+  the criteria for merge (detailed [below](#merge-criteria)).
+- If the review requests changes, it moves into the `reviewed` state. A PR can
+  arrive here either through an assigned reviewer submitting a review, or
+  through a "drive-by" reviewer that requests changes.
+
+Once a PR is in the `reviewed` state, automation will take care of updating
+assignees to reflect whose responsibility the PR is at the present time:
+
+- The reviewer that submitted the review is unassigned.
+- The PR author is assigned to the PR to address the reviewer's feedback.
```

Comment:
Maybe this is implied, but it seems that the PR author should only be re-assigned to the PR if the reviewer did in fact request changes. Another option is the reviewer approved, in which case the re-assignment of the PR author is unnecessary. I think it would be annoying for a PR author to see they are re-assigned to an issue, only to check it and see they have no changes to make.

Reply:
Yeah, it might be helpful to update the wording here. The "reviewed" state really indicates "changes requested" I think.